Client/Frontend/Autofill/CreditCard/CreditCardSettingsViews+Model/CreditCardTableViewController.swift

```diff
@@ -0,0 +1,194 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+import SwiftUI
+import Storage
+import Shared
+
+class CreditCardTableViewController: UIViewController, ThemeApplicable {
+    var viewModel: CreditCardTableViewModel
```

Comment:
The theme observer is missing here and we are not listening for theme changes.

Reply:
Good catch